core/block_render_svg_vertical.js

```diff
@@ -1113,35 +1141,40 @@ Blockly.BlockSvg.prototype.renderClassify_ = function() {
 Blockly.BlockSvg.prototype.renderDrawTop_ =
     function(steps, connectionsXY, rightEdge) {
   /* eslint-disable indent */
-  // Position the cursor at the top-left starting point.
-  if (this.squareTopLeftCorner_) {
-    steps.push('m 0,0');
-    if (this.startHat_) {
-      steps.push(Blockly.BlockSvg.START_HAT_PATH);
-    }
-    // Skip space for the output shape
-    if (this.edgeShapeWidth_) {
-      steps.push('m ' + this.edgeShapeWidth_ + ',0');
-    }
+  if (this.type == Blockly.BlockSvg.DEFINE_BLOCK_TYPE) {
+    steps.push('m 0, 0');
```

Comment:
What does the "move relative 0 0" do? seems like a noop?

Reply:
er yes that looks like a noop.

Reply:
Actually, it's necessary.  All SVG paths have to starts with a move command to say where the drawing starts--it's not set to 0,0 by default.